packages/paste-core/components/form-pill-group/package.json

```diff
@@ -24,6 +24,9 @@
     "clean": "rm -rf ./dist",
     "tsc": "tsc"
   },
+  "dependencies": {
+    "deepmerge": "4.2.2"
```

Comment:
Note: Soooooooooooooooooooo to get this to work and not have super repetitive CSS, I needed to add this as a dependency. Does this make it a major update?